check-updates.sh

```diff
@@ -10,8 +10,8 @@ set -o nounset
 # - curl
 # - dpkg-deb
 
-PREV_ETAG="adca40fbf9f18972d7405c01ff35ef45-15"
-PREV_RELEASE="1.0.9236953-release"
+PREV_ETAG="641a004db2394e96190ef8fd15cfe3c8-16"
+PREV_RELEASE="1.0.9251387~release"
```

Comment:
Hello, I'm trying to understand what is failing. Might it be because of `~` used here instead `-`? Does this affect the build? I am completely new at Flatpak.